layouts/shortcodes/content-tab.html

```diff
@@ -0,0 +1,34 @@
+<!-- 
+     This is a fork from themes/docsy/layouts/shortcodes/tab.html 
+     that allows for using tabs without neccesarily embeding code.
+-->
```

Comment:
It might be possible, but it will mean opening an issue to describe the problem, agree on a solution, then submit a PR and get it merged. Forking it locally seemed like a faster short term solution until I get time to do that.